src/components/views/messages/MLocationBody.tsx

```diff
@@ -147,3 +148,29 @@ export function parseGeoUri(uri: string): GeolocationCoordinates {
         speed: undefined,
     };
 }
+
+function makeLink(coords: GeolocationCoordinates): string {
+    return (
+        "https://www.openstreetmap.org/" +
```

Comment:
can we make this a config.json variable to support self-hosted setups? Not critical for this PR, but something we should do before launch I think.

Reply:
Thanks, I'll add a story.  I guess we'd need to configure the text of the button as well

Reply:
Story created in internal tracker.

Reply:
Button text is probably less of a priority - will copy to internal ticket.